doc/sphinx/userguide/qemusimulation.rst

```diff
@@ -17,7 +17,7 @@ any other target in Freedom E SDK
 
 .. code-block:: bash
 
-   make [PROGRAM=hello] [TARGET=qemu-sifive-e31] [CONFIGURATION=debug] software
+   make PROGRAM=hello TARGET=qemu-sifive-e31 CONFIGURATION=debug software
 
 The square brackets in the above command indicate optional parameters for the
 Make invocation. As you can see, the default values of these parameters tell
```

Comment:
The square brackets where there for a reason. If you remove them you should remove this text underneath explaining what they do.